app/controllers/api/console.php

```diff
@@ -43,8 +43,8 @@
     ))
     ->inject('response')
     ->action(function (Response $response) {
-        $validator = new Domain(System::getEnv('_APP_DOMAIN'));
-        $isDomainValid = !empty(System::getEnv('_APP_DOMAIN', '')) && $validator->isKnown() && !$validator->isTest();
+        $validator = new Domain(System::getEnv('_APP_CONSOLE_DOMAIN', System::getEnv('_APP_DOMAIN', '')));
+        $isDomainValid = !empty(System::getEnv('_APP_CONSOLE_DOMAIN', System::getEnv('_APP_DOMAIN', ''))) && $validator->isKnown() && !$validator->isTest();
```

Comment:
We shouldnt replace all the places, only what's relevant - what's for frontend. Domain validation is very likely for Proxy and DNS.
If we need Console for something on frontend, we can expose that as separate variable